redcap/project.py

```diff
@@ -149,6 +157,423 @@ def _call_api(self, payload, typpe, **kwargs):
         rcr = RCRequest(self.url, payload, typpe)
         return rcr.execute(**request_kwargs)
 
+    def __toggle_dataframe(self,format):
+        ret_format = format
+        if format == 'df':
+            try:
```

Comment:
I like that, it seemed rather awkward as it was. Once I've finished adding methods, I'll try and see if there's anything else that can simplified similarly.